src/style.css

```diff
@@ -7,7 +7,7 @@
 /* This adds some low-level css resets based on the browserslist in package.json, see https://create-react-app.dev/docs/adding-css-reset */
 @import-normalize;
 
-.hover-style:hover {
+.hover-style:hover, .hover-style:focus {
```

Comment:
Separate PR? I think we want these kinds of changes unconditionally, whereas the upload feature has some organizational controversy around it.

Reply:
Thanks for that, good to know. Happy to split it up